src/PythonMigrationViewExtension/MigrationAssistant/PythonMigrationAssistantViewModel.cs

```diff
@@ -8,8 +8,8 @@ namespace Dynamo.PythonMigration.MigrationAssistant
 {
     internal class PythonMigrationAssistantViewModel
     {
-        public string OldCode { get; set; }
-        public string NewCode { get; set; }
+        public string OldCode { get; private set; }
```

Comment:
Sorry guys, I know this is obvious, but it's a public getter, can we please add /// comments.